src/NUnitFramework/tests/Internal/RandomizerTests.cs

```diff
@@ -661,6 +661,25 @@ public void RandomGuidsAreV4()
             Assert.That(bytes[8] & 0xc0, Is.EqualTo(0x80));
         }
 
+#if !NETFRAMEWORK
```

Comment:
This test should not be added.
The whole idea is that optimizations in the library don't affect the tests. The normal test will exercise the new path.